be/app/Http/Controllers/ActivityLogsController.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class ActivityLogsController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $activities = array(
+            [
```

Comment:
- why this one naka static?

Reply:
I asked you about it sir on the meeting, since I still dont have a data coming from user_words activity logs, and you said make a static data for a meantime then go back if okay na ang data

Reply:
do we have separate task for this one on when could this be updated? reply here the asana link :man: